src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What's this for?

Reply:
Weird, looks auto-generated. Not needed.